hydra-node/golden/ChainState.json

```diff
@@ -3,83 +3,84 @@
2466002006004601e6ae854008c014d5d09aba2500223263201d33573802c03a03626aae7940044dd50009191919191999ab9a3370e6aae75401120002333322221233330010050040030023232323333573466e1cd55cea8012400046644246600200600460306ae854008cd404005cd5d09aba2500223263202233573803604404026aae7940044dd50009aba150043335500875ca00e6ae85400cc8c8c8cccd5cd19b875001480108c84888c008010d5d09aab9e500323333573466e1d4009200223212223001004375c6ae84d55cf280211999ab9a3370ea00690001091100191931901219ab9c01d024022021020135573aa00226ea8004d5d0a80119a8063ae357426ae8940088c98c8078cd5ce00b80f00e09aba25001135744a00226aae7940044dd5000899aa800bae75a224464460046eac004c8004d5406488c8cccd55cf80112804119a80399aa80498031aab9d5002300535573ca00460086ae8800c06c4d5d08008891001091091198008020018891091980080180109119191999ab9a3370ea002900011a80398029aba135573ca00646666ae68cdc3a801240044a00e464c6403066ae700440600580544d55cea80089baa0011212230020031122001232323333573466e1d400520062321222230040053007357426aae79400c8cccd5cd19b875002480108c848888c008014c024d5d09aab9e500423333573466e1d400d20022321222230010053007357426aae7940148cccd5cd19b875004480008c848888c00c014dd71aba135573ca00c464c6402c66ae7003c05805004c0480444d55cea80089baa001232323333573466e1cd55cea80124000466442466002006004600a6ae854008dd69aba135744a004464c6402466ae7002c0480404d55cf280089baa0012323333573466e1cd55cea800a400046eb8d5d09aab9e500223263201033573801202001c26ea80048c8c8c8c8c8cccd5cd19b8750014803084888888800c8cccd5cd19b875002480288488888880108cccd5cd19b875003480208cc8848888888cc004024020dd71aba15005375a6ae84d5d1280291999ab9a3370ea00890031199109111111198010048041bae35742a00e6eb8d5d09aba2500723333573466e1d40152004233221222222233006009008300c35742a0126eb8d5d09aba2500923333573466e1d40192002232122222223007008300d357426aae79402c8cccd5cd19b875007480008c848888888c014020c038d5d09aab9e500c23263201933573802403202e02c02a02802602402226aae7540104d55cf280189aab9e5002135573ca00226ea80048c8c8c8c8cccd5cd19b875001480088ccc888488ccc00401401000cdd69aba15004375a6ae85400cdd69aba135744a00646666ae68cdc3a80124000464244600400660106ae84d55cf280311931900919ab9c00b01201000f135573aa00626ae8940044d55cf280089baa001232323333573466e1d400520022321223001003375c6ae84d55cf280191999ab9a3370ea004900011909118010019bae357426aae7940108c98c803ccd5ce00400780680609aab9d50011375400224464646666ae68cdc3a800a40084244400246666ae68cdc3a8012400446424446006008600c6ae84d55cf280211999ab9a3370ea00690001091100111931900819ab9c00901000e00d00c135573aa00226ea80048c8cccd5cd19b8750014800880208cccd5cd19b8750024800080208c98c8030cd5ce00280600500489aab9d3754002921035054310023500549011b65787065637465642073696e676c652068656164206f75747075740023500449012077726f6e67207175616e74697479206f66205054206469737472696275746564001220021220011232632003335738002006930900088919180080091198019801001000a4411cbbed691fc7fdce8d95616cf16b01e5ed2abb42f20c0a5de6882bb8600048811cb16266c24ccb6af2d4201d511d41caeeb2d785a0531be26db492f2a6003351223300248920c583a039031a9ff40605d7518691608267248c42e2da459d6417780da60a084b00480f08848cc00400c0088005",
```

Comment:
Should: not re-generate golden files as we do not need to update our state types (see other comments).

Reply:
Don't we need to keep contesters in the `ClosedThreadOutput` in order to construct the expected output datum offchain?

Reply:
Yes. You're right. this golden needs to change, yes.